platforms/android/README.md

```diff
@@ -0,0 +1,54 @@
+Android
+=======
+
+The recommended way use tangram-es in an Android project is to add it as a Gradle dependency. The library is hosted on Maven Central in the package 'com.mapzen.tangram'. To find the latest version number, check [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.mapzen.tangram%22).
+
+If Maven Central is not set up as a dependency repository, add it in your project's 'build.gradle' file:
+
+```
+allprojects {
+  dependencies {
+    repositories {
+      mavenCentral()
+    }
+  }
+}
+```
+
+Then add tangram-es in the 'dependencies' section of your module's 'build.gradle' file:
+
+```
+dependencies {
+  compile 'com.mapzen.tangram:tangram:0.5.0'
+}
+```
+
+That's it! If you want to build tangram-es for Android from scratch, continue reading.
+
+## Setup ##
+
+To build for Android you'll need [Android Studio](https://developer.android.com/studio/index.html) version 2.2 or newer on Mac OS X, Ubuntu, or Windows 10. Using the Android Studio SDK Manager, install or update the 'CMake', 'LLDB', and 'NDK' packages from the 'SDK Tools' tab.
+
+## Build ##
+
+After installing dependencies in Android Studio, you can execute Android builds from either the command line or the Android Studio interface.
+
+### Command Line ###
+
+The Android project is executed with Gradle commands from the 'platforms/android' folder. To build the demo application for the ARMv7 architecture (covers most Android devices), run:
```

Comment:
Probably make sense to mention about ndk and sdk path settings in relevant file if one is straight away trying from command line.